deploy/helm/flightctl/values.yaml

```diff
@@ -64,6 +69,11 @@ global:
     tls: 443
     http: 80
 
+## @section ACM operator specific parameters
+hubConfig:
+  ocpIngress: ""
+  apiUrl: ""
+
```

Comment:
I think these are not needed anymore ? As we moved it under `global` section

Reply:
True, this is a leftover. Thanks!